apps/backoffice-v2/src/pages/Entity/components/CallToAction/CallToAction.tsx

```diff
@@ -45,112 +47,224 @@ export const CallToAction: FunctionComponent<ICallToActionProps> = ({ value, dat
     workflowLevelResolution,
   } = useCallToActionLogic();
 
-  return value === 'Re-upload needed' ? (
-    <Dialog>
-      <AnimatePresence>
-        <DialogTrigger asChild>
-          <MotionButton
-            {...motionProps}
-            size="wide"
-            variant="warning"
-            disabled={!caseState.actionButtonsEnabled || data?.disabled}
-          >
-            {value}
-          </MotionButton>
-        </DialogTrigger>
-      </AnimatePresence>
-      <DialogContent className={`mb-96`}>
-        <DialogHeader>
-          <DialogTitle className={`text-2xl`}>Mark document for re-upload</DialogTitle>
-          <DialogDescription asChild>
-            <p className="text-sm">
-              {workflowLevelResolution ? (
-                `Once marked, you can use the “Ask for all re-uploads” button at the top of the
-                  screen to initiate a request for the customer to re-upload all of the documents
-                  you have marked for re-upload.`
-              ) : (
-                <>
-                  <span className="mb-[10px] block">
-                    By clicking the button below, an email with a link will be sent to the customer,
-                    directing them to re-upload the documents you have marked as “re-upload needed”.
-                  </span>
-                  <span>
-                    The case’s status will then change to “Revisions” until the customer will
-                    provide the needed documents and fixes.
-                  </span>
-                </>
-              )}
-            </p>
-          </DialogDescription>
-        </DialogHeader>
-        {!noReasons && (
+  if (value === 'Reject') {
```

Comment:
Pasted the dialog from the `dev` branch and updated the buttons to fit the new design direction.